### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zli82016**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+* compute: promoted `google_compute_region_instance_template` to GA ([#15710](https://github.com/hashicorp/terraform-provider-google/pull/15710))
+* container: added additional options for field `monitoring_config.enable_components` in `google_container_cluster` resource ([#15727](https://github.com/hashicorp/terraform-provider-google/pull/15727))
+* gkehub: added `labels` field to `google_gke_hub_scope_rbac_role_binding` resource ([#15729](https://github.com/hashicorp/terraform-provider-google/pull/15729))
+* logging: added in-place update support for field `unique_writer_identity` in `google_logging_project_sink` resource ([#15721](https://github.com/hashicorp/terraform-provider-google/pull/15721))
+* networkconnectivity: added `psc_connections.error.details` field to `google_network_connectivity_service_connection_policy` resource ([#15726](https://github.com/hashicorp/terraform-provider-google/pull/15726))
+* secretmanager: added in-place update support for field `replication.user_managed.replicas.customer_managed_encryption` field in `google_secret_manager_secret` resource ([#15685](https://github.com/hashicorp/terraform-provider-google/pull/15685))
```
Comment (shuyama1) [inline CHANGELOG.md:12]:
```suggestion
* secretmanager: added in-place update support for field `replication.user_managed.replicas.customer_managed_encryption` in `google_secret_manager_secret` resource ([#15685](https://github.com/hashicorp/terraform-provider-google/pull/15685))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,54 @@
-## 4.81.0 (Unreleased)
+## 4.82.0 (Unreleased)
+
+IMPROVEMENTS:
+* compute: added update support for field `enable_proxy_protocol` in `google_compute_service_attachment` resource ([#15716](https://github.com/hashicorp/terraform-provider-google/pull/15716))
+* compute: added update support for field `reconcile_connections` in `google_compute_service_attachment` resource ([#15706](https://github.com/hashicorp/terraform-provider-google/pull/15706))
+* compute: added update support for field `allowPscGlobalAccess` in `google_compute_forwarding_rule` resource ([#15691](https://github.com/hashicorp/terraform-provider-google/pull/15691))
```
Comment (shuyama1) [inline CHANGELOG.md:6]:
```suggestion
* compute: added in-place update support for field `enable_proxy_protocol` in `google_compute_service_attachment` resource ([#15716](https://github.com/hashicorp/terraform-provider-google/pull/15716))
* compute: added in-place update support for field `reconcile_connections` in `google_compute_service_attachment` resource ([#15706](https://github.com/hashicorp/terraform-provider-google/pull/15706))
* compute: added in-place update support for field `allowPscGlobalAccess` in `google_compute_forwarding_rule` resource ([#15691](https://github.com/hashicorp/terraform-provider-google/pull/15691))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,54 @@
...
+* logging: added in-place update support for field `unique_writer_identity` in `google_logging_project_sink` resource ([#15721](https://github.com/hashicorp/terraform-provider-google/pull/15721))
+* networkconnectivity: added `psc_connections.error.details` field to `google_network_connectivity_service_connection_policy` resource ([#15726](https://github.com/hashicorp/terraform-provider-google/pull/15726))
+* secretmanager: added in-place update support for field `replication.user_managed.replicas.customer_managed_encryption` field in `google_secret_manager_secret` resource ([#15685](https://github.com/hashicorp/terraform-provider-google/pull/15685))
+
+BUG FIXES:
+* bigquery: fixed an error from updating immutable `params` of `google_bigquery_data_transfer_config` ([#15723](https://github.com/hashicorp/terraform-provider-google/pull/15723))
+* dns: fixed hash function for `network_url` in `google_dns_managed_zone` and `google_dns_policy` resources ([#15728](https://github.com/hashicorp/terraform-provider-google/pull/15728))
```
Comment (shuyama1) [inline CHANGELOG.md:16]: I wonder if we can change it a bit to be more user-focused

File: CHANGELOG.md (hunk 1 above)
Comment (zli82016) [reply CHANGELOG.md]: Good catch. Thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (zli82016) [reply CHANGELOG.md]: Changed.

File: CHANGELOG.md (hunk 3 above)
Comment (zli82016) [reply CHANGELOG.md]: Yeah. Add more details.

**Review by shuyama1 — APPROVED** [review]

**Pull request merged**